packages/react/src/progress/Progress.css.ts

```diff
@@ -0,0 +1,38 @@
+import { theme } from "@optiaxiom/globals";
+
+import { recipe, type RecipeVariants, style } from "../vanilla-extract";
+
+export const progress = recipe({
+  base: [
+    {
+      h: "full",
+    },
+    style({
+      transition: "all 400ms",
+    }),
+  ],
+  variants: {
+    fullWidth: {
+      false: style({
+        borderBottomLeftRadius: theme.borderRadius.full,
+        borderTopLeftRadius: theme.borderRadius.full,
+      }),
+      true: style({
+        borderRadius: theme.borderRadius.full,
+      }),
+    },
+    intent: {
+      default: {
+        bg: "bg.accent",
+      },
+      error: {
```

Comment:
Should be called danger
```suggestion
      danger: {
```